run_service.sh

```diff
@@ -683,25 +648,18 @@ suggested_safe_top_up_default=500000000000000000
 
 export RPC_RETRIES=40
 export RPC_TIMEOUT_SECONDS=120
+
+# export CUSTOM_SERVICE_REGISTRY_ADDRESS="0x9338b5153AE39BB89f50468E608eD9d764B755fD"  # Set up in .env file
+# export CUSTOM_STAKING_ADDRESS="0x43fB32f25dce34EB76c78C7A42C8F40F84BCD237"  # Set up in .env file
+# export CUSTOM_OLAS_ADDRESS="0xcE11e14225575945b8E6Dc0D4F2dD4C570f79d9f"  # Set up in .env file
+# export CUSTOM_SERVICE_REGISTRY_TOKEN_UTILITY_ADDRESS="0xa45E64d13A30a51b91ae0eb182e88a40e9b18eD8"  # Set up in .env file
+# export MECH_CONTRACT_ADDRESS="0x77af31De935740567Cf4fF1986D04B2c964A786a"  # Set up in env file
```

Comment:
@kupermind Please validate the following:
1. The parameters commented correspond to addresses that depend/can be inferred from the staking contract.
2. The remaining addresses not commented should remain constant for all staking contracts, for the time being.
3. Please suggest a more appropriate name for any of these variables from point 1 and 2 above.

Reply:
CUSTOM_GNOSIS_SAFE_PROXY_FACTORY_ADDRESS => CUSTOM_GNOSIS_MULTISIG_ADDRESS

Reply:
These commented variables will be removed before merging to main.